### DISCUSSION TITLE
Add Sonic Robo Blast 2

### DISCUSSION THREAD

**Discussion by aWhhdGVteXNlbGY** [tltneon/lgsl#177] [Ideas]
Some sources for references:
https://ms.srb2.org/
https://git.do.srb2.org/Golden/SRB2-Query/tree/update-to-2.2.1 (an PHP example code, although I feel it's outdated)
https://wiki.srb2.org/wiki/PT_SERVERINFO

**Comment by tltneon** [comment]
Hi, @ThatRoboticFish 
I added simple variant of protocol (w/o players), you may try it

**Reply by aWhhdGVteXNlbGY** [reply]
Yeah it's working nice, thanks. Hope you can add connected players soon.

**Reply by aWhhdGVteXNlbGY** [reply]
BTW, one thing I noticed is that when passing the mouse cursor over the server hostname it also displays undefined characters as padding
![imagen](https://github.com/tltneon/lgsl/assets/113400974/3a74172a-0a0d-4f4a-b515-ec39c06c875c)
![imagen](https://github.com/tltneon/lgsl/assets/113400974/584b6eef-871a-425d-9c10-458f0d201ecc)
